connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -695,6 +705,32 @@ ErrorHandlingMetrics errorHandlingMetrics(ConnectorTaskId id) {
         return reporters;
     }
 
+    private WorkerErrantRecordReporter createWorkerErrantRecordReporter(
+        ConnectorTaskId id,
+        SinkConnectorConfig connConfig,
+        Class<? extends Connector> connectorClass,
+        Converter keyConverter,
+        Converter valueConverter,
+        HeaderConverter headerConverter
+    ) {
+        // check if errant record reporter topic is configured
+        String topic = connConfig.dlqTopicName();
+        if ((topic != null && !topic.isEmpty()) || connConfig.enableErrorLog()) {
```

Comment:
would be nice if this was a method in the config

Reply:
Nice idea, although unfortunately there were no new configs introduced in the KIP and I cannot add this to the implementation if it was not apart of the voted on design.

Reply:
The `SinkConnectorConfig` class isn't part of the public API; we can modify it without a KIP if we want.